### ISSUE TITLE
Report status default

### ISSUE THREAD

**Issue by hglkrijger** [Azure/WALinuxAgent#1169]
We should switch the report status default to HostGAPlugin, as access to managed storage accounts will soon be locked down.

**Comment by boumenot** [comment]
I believe it is sufficient to make this change only.

```python
class HostPluginProtocol(object):
    _is_default_channel = True
```

**Comment by hglkrijger** [comment]
Unfortunately not - we only want HostGA to be default for reporting status, not for fetching artifacts.

**Comment by boumenot** [comment]
This was fixed in #1194.
